car_framework/car_service.py

```diff
@@ -113,38 +112,24 @@ def check_import_status(self, statuses):
             for s in jobs_to_check.values():
                 s.status = ImportJobStatus.FAILURE
                 s.error = str(e)
-        
+
 
     def delete(self, resource, ids):
         # report and source not mentioned anywhere coz connectors aren't allowed to delete it
         key_based = ["ipaddress", "hostname", "macaddress"]
         external_id_based = ["asset", "container", "user", "account", "application", "database", "port", "vulnerability", "geolocation"]
-        
         if resource in key_based:
-            resource_key = 'keys'
```

Comment:
Seems you overrode my changes in delete function:
https://github.com/IBM/cp4s-car-connector-framework/pull/5/files#diff-213769a12cf7317bbd297d6d5d37ffed47b1f02ab8dc826bfc789fcac85a8916L123
was there any specific reason for that or it is merge problem?

Reply:
Right, merge problem. Let me fix it...

Reply:
Fixed